### ISSUE TITLE
Not Deployable

### ISSUE THREAD

**Issue by EmilioGalicia** [financialforcedev/fflib-apex-common#196]
Deploy to Salesforce not working, fails with the following log:

Deployment Started
Status: Queued 
Status: InProgress 
Status: Completed 
Deployment Complete
Failures:
classes/fflib_ApplicationTest.cls(295,3):Variable does not exist: mocks
classes/fflib_ApplicationTest.cls(314,3):Invalid type: fflib_ApexMocks
classes/fflib_ApplicationTest.cls(315,3):Variable does not exist: mocks
classes/fflib_ApplicationTest.cls(317,22):Variable does not exist: fflib_IDGenerator
classes/fflib_ApplicationTest.cls(318,26):Variable does not exist: fflib_IDGenerator
classes/fflib_ApplicationTest.cls(325,3):Variable does not exist: mocks
classes/fflib_ApplicationTest.cls(118,3):Variable does not exist: mocks
classes/fflib_ApplicationTest.cls(120,22):Variable does not exist: fflib_IDGenerator
classes/fflib_ApplicationTest.cls(127,3):Variable does not exist: mocks
classes/fflib_ApplicationTest.cls(128,3):Variable does not exist: mocks
classes/fflib_ApplicationTest.cls(129,3):Variable does not exist: mocks
classes/fflib_ApplicationTest.cls(211,63):Invalid type: fflib_ApexMocks
classes/fflib_ApplicationTest.cls(271,6):Variable does not exist: fflib_IDGenerator
classes/fflib_ApplicationTest.cls(272,6):Variable does not exist: fflib_IDGenerator
classes/fflib_ApplicationTest.cls(283,3):Invalid type: fflib_ApexMocks
classes/fflib_ApplicationTest.cls(284,3):Variable does not exist: mocks
classes/fflib_ApplicationTest.cls(286,22):Variable does not exist: fflib_IDGenerator
classes/fflib_ApplicationTest.cls(293,3):Variable does not exist: mocks
classes/fflib_ApplicationTest.cls(294,3):Variable does not exist: mocks
classes/fflib_ApplicationTest.cls(35,22):Variable does not exist: fflib_IDGenerator
classes/fflib_ApplicationTest.cls(46,19):Variable does not exist: fflib_IDGenerator
classes/fflib_ApplicationTest.cls(58,26):Variable does not exist: fflib_IDGenerator
classes/fflib_ApplicationTest.cls(70,23):Variable does not exist: fflib_IDGenerator
classes/fflib_ApplicationTest.cls(82,3):Invalid type: fflib_ApexMocks
classes/fflib_ApplicationTest.cls(83,3):Variable does not exist: mocks
classes/fflib_ApplicationTest.cls(85,3):Variable does not exist: mocks
classes/fflib_ApplicationTest.cls(86,3):Variable does not exist: mocks
classes/fflib_ApplicationTest.cls(117,3):Invalid type: fflib_ApexMocks
classes/fflib_ApplicationTest.cls(326,3):Variable does not exist: mocks
classes/fflib_ApplicationTest.cls(327,3):Variable does not exist: mocks
classes/fflib_SObjectMocks.cls(7,11):Invalid type: fflib_ApexMocks
classes/fflib_SObjectMocks.cls(9,40):Invalid type: fflib_ApexMocks
classes/fflib_SObjectMocks.cls(16,32):Variable does not exist: mocks
classes/fflib_SObjectMocks.cls(21,27):Variable does not exist: mocks
classes/fflib_SObjectMocks.cls(27,11):Invalid type: fflib_ApexMocks
classes/fflib_SObjectMocks.cls(29,42):Invalid type: fflib_ApexMocks
classes/fflib_SObjectMocks.cls(36,32):Variable does not exist: mocks
classes/fflib_SObjectMocks.cls(41,27):Variable does not exist: mocks
classes/fflib_SObjectMocks.cls(47,11):Invalid type: fflib_ApexMocks
classes/fflib_SObjectMocks.cls(49,44):Invalid type: fflib_ApexMocks
classes/fflib_SObjectMocks.cls(56,4):Variable does not exist: mocks
classes/fflib_SObjectMocks.cls(61,4):Variable does not exist: mocks
classes/fflib_SObjectMocks.cls(66,4):Variable does not exist: mocks
classes/fflib_SObjectMocks.cls(71,4):Variable does not exist: mocks
classes/fflib_SObjectMocks.cls(76,4):Variable does not exist: mocks
classes/fflib_SObjectMocks.cls(81,4):Variable does not exist: mocks
classes/fflib_SObjectMocks.cls(86,4):Variable does not exist: mocks
classes/fflib_SObjectMocks.cls(95,13):Variable does not exist: mocks
classes/fflib_SObjectMocks.cls(100,4):Variable does not exist: mocks
classes/fflib_SObjectMocks.cls(105,4):Variable does not exist: mocks
classes/fflib_SObjectMocks.cls(110,4):Variable does not exist: mocks
classes/fflib_SObjectMocks.cls(115,4):Variable does not exist: mocks
classes/fflib_SObjectMocks.cls(120,4):Variable does not exist: mocks
classes/fflib_SObjectMocks.cls(125,4):Variable does not exist: mocks

**Comment by stohn777** [comment]
Did you deploy Apex Mocks?

Notice the statement on the home page.
Dependencies: Must deploy ApexMocks before deploying this library.

https://github.com/financialforcedev/fflib-apex-mocks

**Comment by EmilioGalicia** [comment]
That must be the reason, let me try again

**Comment by agarciaffdc** [comment]
@EmilioGalicia, please, let us know if you are still having issues. Thanks!!

**Comment by afawcett** [comment]
@agarciaffdc i think given the time thats passed we should close this now. The @EmilioGalicia can still comment on this and it can be re-opened if he is still having issues. 👍
